### PR TITLE
Added support for center crop and resizing

### PR DESCRIPTION
**Pull request by kwotsin**
I've added some support for performing central crop with a specific fraction, since this is a common preprocessing technique used in ImageNet classification (2012) validation preprocessing. I have tested the performance with **Inception V3** pretrained and with the preprocessing transformation seen below, I can get a validation accuracy of up till **77.61% for all 50k images**, which is close to the original reported **78.0% accuracy from Google**. This should be close enough for the sake of reproducibility. 

```
transform = transforms.Compose([
    transforms.CenterResizedCrop(299),
    transforms.ToTensor(),  
])
```
On top of the transformation above, I also needed to perform something as simple as `(input - 0.5)  * 2.0` in my data loading, in order to match the preprocessing operation used by google as seen here:
https://github.com/tensorflow/models/blob/master/research/slim/preprocessing/inception_preprocessing.py#L243-L281

This got the val accuracy up by another 0.3%. I believe the remaining difference in val accuracy could be attributed to training differences between the 2 frameworks.

----
### Regarding changes
To prevent breaking programs using existing function, I created the new `resized_center_crop` function instead of editing the RandomResizedCrop (which can technically achieve the same thing, although you would have to ignore many paramters and set the scale to a tight bound). I think this will be a nice addition to researchers working on the ImageNet dataset esp. when they want to replicate the performance seen in TensorFlow.

Please check if there are any issues with this pull request -- I'll figure a way to fix any issues as soon as I can.

### CODE REVIEW

File: torchvision/transforms/transforms.py [hunk 1]
```diff
@@ -199,7 +199,9 @@ class CenterCrop(object):
     """
```
Comment (SsnL) [inline torchvision/transforms/transforms.py]: Doc above needs to be updated. And what about `(float, float)`?

Comment (kwotsin) [reply torchvision/transforms/transforms.py]: Just found the duplicated function and have made the changes. As for the (float, float) case, I didn't include it initially as I didn't think there would be a use for it, but greater flexibility is probably a good idea. I have made the changes. Tested with `(float, float)` input using `(0.875, 0.875)` and got 77.61% validation accuracy again. Waiting to see if the CI checks pass again, but there's a possible mutual import error from importing transforms (can you see if there is one?).

**Comment by Coderx7** [comment]
@kwotsin : First of all thanks a lot for your efforts. I would like to use your changes as well. 
based on your latest changes, what do I need to change ? 
Can you share your training/validation transformations as well ?
